### PR TITLE
Papertrail logspout

### PR DESCRIPTION
**Pull request by copelco**
Adds [Log from the standard Docker streams](https://help.papertrailapp.com/kb/configuration/configuring-centralized-logging-from-kubernetes/) from Papertrail docs.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -8,3 +8,6 @@ k8s_letsencrypt_email: ''
 k8s_echotest_state: present
 k8s_echotest_hostname: 'echotest.caktus-built.com'
 k8s_echotest_letsencrypt_issuer: 'letsencrypt-staging'
+
+k8s_papertrail_logspout_namespace: 'default'
+k8s_papertrail_logspout_destination: ''
```
Comment (vkurup) [inline defaults/main.yml]: Not sure if we are providing documentation hints here, or elsewhere, but might be nice to say that this should look like `syslog+tls://logsN.papertrailapp.com:XXXXX`

File: templates/papertrail/papertrail-logspout-daemonset.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: logspout-papertrail-secrets
+  namespace: '{{ k8s_papertrail_logspout_namespace }}'
```
Comment (vkurup) [inline templates/papertrail/papertrail-logspout-daemonset.yml:5]: if we allow customization of the namespace, do we have to add a block to create that namespace?

**Review by vkurup — APPROVED** [review]
A few comments (that probably reflect my k8s ignorance more than anything), but this looks like it lines up with https://help.papertrailapp.com/kb/configuration/configuring-centralized-logging-from-kubernetes/#log-from-the-standard-docker-streams and https://help.papertrailapp.com/assets/files/papertrail-logspout-daemonset.yml

:shipit:

File: defaults/main.yml (hunk 1 above)
Comment (copelco) [reply defaults/main.yml]: Good idea!

File: templates/papertrail/papertrail-logspout-daemonset.yml (hunk 2 above)
Comment (copelco) [reply templates/papertrail/papertrail-logspout-daemonset.yml:5]: Good point. I added a comment in defaults.yml to note this.

**Pull request merged**